app/controllers/textile_controller.rb

```diff
@@ -7,6 +7,11 @@ def field
   # Returns the form view given a source text
   def form
     @form_data = Note.parse_fields(params[:source]) if params[:source]
+
+    if fieldless_string && !fieldless_string.empty?
```

Comment:
Is it okay if we use `.present?` instead of double negation of `!empty?`? `.present?` seems more intuitive